assignment/assignment-tool/tool/src/webapp/vm/assignment/assignment_macros.vm

```diff
@@ -37,12 +37,12 @@
 ## Defining macro to repeat buttons at the top and bottom of the page
 #macro( buttonBar $submitnotif )
     <div class="act">
-        <input accesskey="s" type="button" class="active" name="post" value="$tlang.getString("gen.pos")" onclick="SPNR.disableControlsAndSpin( this, null ); ASN.submitForm( 'newAssignmentForm', 'post', null, null, true );" />
-        <input accesskey="v" type="button" name="preview" value="$tlang.getString("gen.pre")" onclick="SPNR.disableControlsAndSpin( this, null ); ASN.submitForm( 'newAssignmentForm', 'preview', null, null, true );" />
+        <input accesskey="s" type="button" class="active" id="post" name="post" value="$tlang.getString("gen.pos")" onclick="SPNR.disableControlsAndSpin( this, null ); ASN.submitForm( 'newAssignmentForm', 'post', null, null, true );" />
```

Comment:
Won't this result in the same ID appearing multiple times on the page (which shouldn't happen) because the button bar is repeated at the top and bottom.